lib/constants/framework-initializer.constants.js

```diff
@@ -1,150 +1,159 @@
+  const fullPath = `templates/${subdirectory}`;
+
+  return `
+mkdir ${projectName}
+cd ${projectName}
+git init
+git remote add -f origin https://github.com/aziontech/vulcan-examples.git > /dev/null 2>&1
+git config core.sparseCheckout true
+echo "${fullPath}/*" >> .git/info/sparse-checkout
+git pull origin main > /dev/null 2>&1
+mv ${fullPath}/* ./
+rm -rf templates .git
+`.trim();
+};
+
+const TemplatesOptions = {
+  JavaScript: {
+    options: [
+      {
+        value: 'javascript/simple-js-router',
+        message: 'Simple JS Router',
+        mode: 'compute',
+      },
+      {
+        value: 'javascript/simple-js-node',
+        message: 'Simple Node.js with Polyfills',
+        mode: 'compute',
+      },
+    ],
+  },
+  TypeScript: {
+    options: [{ value: 'typescript/simple-ts-router', mode: 'compute' }],
+  },
   Angular: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'angular-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Astro: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'astro-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Gatsby: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'gatsby-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Hexo: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'hexo-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Next: {
     options: [
-      { value: '12.3.1', message: 'SSR supported' },
-      { value: 'latest', message: 'static supported' },
+      {
+        value: 'next-13-static',
+        message: 'Next 13 - Static (export)',
+        mode: 'deliver',
+      },
+      {
+        value: 'nextjs/edge-app-13-5-6',
+        message: 'Next 13 - Edge Runtime (App Router) ',
+        mode: 'compute',
+      },
+      {
+        value: 'nextjs/edge-pages-13-5-6',
+        message: 'Next 13 - Edge Runtime (Pages Router)',
+        mode: 'compute',
+      },
+      {
+        value: 'nextjs/node-pages-12-3-1',
+        message: 'Next 12 - Node Runtime (Edge compatible with polyfills)',
+        mode: 'compute',
+      },
     ],
   },
   React: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'react-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Vue: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      { value: 'vue-static', message: 'static supported', mode: 'deliver' },
+    ],
   },
   Vite: {
-    options: [{ value: 'latest', message: 'static supported' }],
+    options: [
+      {
+        value: 'vue3-vite-static',
+        message: 'static supported',
+        mode: 'deliver',
+      },
+    ],
+  },
+  'C++': {
```

Comment:
We must change the name to **Emscripten**. You can use multiple languages to code. From the docs:
`Compile your existing projects written in C or C++ — or any language that uses LLVM — to browsers, Node.js, or wasm runtimes.`